### PR TITLE
Ctcp 731

### PR DESCRIPTION
**Pull request by HelenECallaghan**

### CODE REVIEW

**Review by PeterBandura — APPROVED** [review]

**Review by crystalmark — APPROVED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Review by fmcgrath — APPROVED** [review]
Looks OK and link works.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

File: source/documentation/index.md [hunk 1]
```diff
@@ -18,20 +18,16 @@ For technical information, see the [CTC Traders API specifications](/api-documen
...
-    Currently, the IE015 message is the only one that can be tested, using the departure
-endpoint. Other messages will be supported later as the service develops. For this
-reason, these XSDs are issued with the proviso they may be subject to change and
-iterated in the future.
-
+The declaration departure endpoint now accepts messages in JSON format as well as XML format. The JSON schemas are available for download [here](https://github.com/hmrc/transit-movements-validator/tree/main/conf/json).
 
 ### What are we working on now?
-Currently, we are working on enabling the saving of departure declaration information. This allows developers to retrieve departure declaration metadata.
+Currently, we are working on retrieving messages.
```
Comment (drnaylor-oc) [inline source/documentation/index.md:24]: Perhaps "departure metadata and associated messages"? 

I think we need to specify "departures" in here, at least, as that's all we're working on right now.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Pull request merged**